run-traffic.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+BASE_URL='https://eagle-test.pathfinder.gov.bc.ca'
+PROJECT_PREFIX='p/58851197aaecd9001b8227cc/project-details;currentPage=1;pageSize=10;sortBy=-dateAdded;ms='
+PROJECTS=(1580750409346 1580750409346 1580750585604 1580750713545)
```

Comment:
convert this to a PATH variable per the ticket

Reply:
Done